exchanges/binanceus/binanceus_wrapper.go

```diff
@@ -0,0 +1,975 @@
+			&CancelOrderRequestParams{
+				Symbol:                o.Pair,
+				OrderID:               o.OrderID,
+				ClientSuppliedOrderID: o.ClientOrderID,
+			})
+		if err != nil {
+			return err
+		}
+	} else {
+		return fmt.Errorf("%w '%v'", asset.ErrNotSupported, o.AssetType)
+	}
+	return nil
+}
+
+// CancelBatchOrders cancels orders by their corresponding ID numbers
+func (bi *Binanceus) CancelBatchOrders(ctx context.Context, orders []order.Cancel) (order.CancelBatchResponse, error) {
+	return order.CancelBatchResponse{}, common.ErrNotYetImplemented
+}
+
+// CancelAllOrders cancels all orders associated with a currency pair
+func (bi *Binanceus) CancelAllOrders(ctx context.Context, orderCancellation *order.Cancel) (order.CancelAllResponse, error) {
+	if err := orderCancellation.Validate(); err != nil {
+		return order.CancelAllResponse{}, err
+	}
+	var cancelAllOrdersResponse order.CancelAllResponse
+	cancelAllOrdersResponse.Status = make(map[string]string)
+	if orderCancellation.AssetType == asset.Spot {
+		symbolValue, err := bi.FormatSymbol(orderCancellation.Pair, asset.Spot)
+		if err != nil {
+			return cancelAllOrdersResponse, err
+		}
+		openOrders, er := bi.GetAllOpenOrders(ctx, symbolValue)
+		if er != nil {
+			return cancelAllOrdersResponse, er
+		}
+		for ind := range openOrders {
+			pair, er := currency.NewPairFromString(openOrders[ind].Symbol)
+			if er != nil {
+				return cancelAllOrdersResponse, er
+			}
+			_, err := bi.CancelExistingOrder(ctx, &CancelOrderRequestParams{
+				Symbol:                pair,
+				OrderID:               strconv.FormatUint(openOrders[ind].OrderID, 10),
+				ClientSuppliedOrderID: openOrders[ind].ClientOrderID,
+			})
+			if err != nil {
+				return cancelAllOrdersResponse, err
+			}
+		}
+	} else {
+		return cancelAllOrdersResponse, fmt.Errorf("%w '%v'", asset.ErrNotSupported, orderCancellation.AssetType)
+	}
+	return cancelAllOrdersResponse, nil
+}
+
+// GetOrderInfo returns order information based on order ID
+func (bi *Binanceus) GetOrderInfo(ctx context.Context, orderID string, pair currency.Pair, assetType asset.Item) (order.Detail, error) {
+	var respData order.Detail
+	orderIDInt, err := strconv.ParseInt(orderID, 10, 64)
+	if err != nil {
+		return respData, err
+	}
+	symbolValue, err := bi.FormatSymbol(pair, asset.Spot)
+	if err != nil {
+		return respData, err
+	}
+	if assetType == asset.Spot {
+		resp, err := bi.GetOrder(ctx, &OrderRequestParams{
+			Symbol:            symbolValue,
+			OrderID:           uint64(orderIDInt),
+			OrigClientOrderID: "",
+		})
+		if err != nil {
+			return respData, err
+		}
+		orderSide, err := order.StringToOrderSide(resp.Side)
+		if err != nil {
+			log.Errorf(log.ExchangeSys, "%s %v", bi.Name, err)
+		}
+		status, err := order.StringToOrderStatus(resp.Status)
+		if err != nil {
+			log.Errorf(log.ExchangeSys, "%s %v", bi.Name, err)
+		}
+		orderType := order.Limit
+		if strings.EqualFold(resp.Type, "MARKET") {
+			orderType = order.Market
+		}
```

Comment:
You can expand the amount of order types handled by using
```suggestion
		orderType, err := order.StringToOrderType(resp.Type)
		if err != nil {
			log.Errorf(log.ExchangeSys, "%s %v", bi.Name, err)
		}
```